MDANSE/Src/MDANSE/Trajectory/AtomicTrajectory.py

```diff
@@ -0,0 +1,13 @@
+import numpy as np
+
+
+def atomic_trajectory(config, cell, rcell, box_coordinates=False):
+    trajectory = np.empty((config.shape[0], 3), dtype=np.float64)
```

Comment:
I think this is slightly different to the cython code. Also could you add a numpy docstring and type hints to this?